src/Field.ts

```diff
@@ -943,13 +967,14 @@ export class Field {
 
                 const message = "Node " + node.getKey() + " (" + node.getName() + ") with category " + node.getCategory() + " contains field (" + field.getDisplayText() + ") with unsuitable type (" + field.getParameterType() + ").";
                 const issue: Errors.Issue = Errors.ShowFix(message, function(){Utils.showField(eagle, node.getId(),field);}, function(){Utils.fixFieldParameterType(eagle, node, field, suitableType)}, "Switch to suitable type, or remove if no suitable type");
-                errorsWarnings.warnings.push(issue);
+                field.issues().push({issue:issue,validity:Errors.Validity.Warning})
+                // errorsWarnings.warnings.push(issue);
             }
         }
 
-        field.errorsWarnings(errorsWarnings)
+        // field.errorsWarnings(errorsWarnings)
 
-        return errorsWarnings
+        // return errorsWarnings
```

Comment:
I think you can remove all the commented code in this function.

All the "errorsWarnings.warnings.push()" and "errorsWarning.errors.push()" etc throughout the function. And these two commented lines at the end.